logging/bin/getlogs.py

```diff
@@ -95,6 +103,21 @@ def validate_input(dict):
             print("Please remove single quotes ('') from search argument.")
             sys.exit()
 
+def open_port(dict):
+    """Binds the v4m-search service on port 9200 to a locally available port"""
+    #Get open port
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s.bind(("",0))
+    s.listen(1)
+    port = s.getsockname()[1]
+    s.close()
+
+    cmd = (["kubectl", "-n", "logging", "port-forward", "svc/v4m-search", str(port) + ':9200', '&'])
```

Comment:
This contains a  hard-coded reference to the "logging" namespace..  While that is the default name for the namespace in which the Viya 4 Monitoring log monitoring components are deployed, users can use something else.  Therefore, this script should find the namespace in which the v4m-search service is deployed and use it here.  Sorry, I suspect that detail was probably left out of the requirements.

In addition, what happens if this command fails?  Do we need logic detecting the failure?  Or, will it fail in a way that makes it clear to the user what happened?  I suspect it doesn't.